addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/JsFunctionScanRuleUnitTest.java

```diff
@@ -76,7 +76,7 @@ void shouldAlertGivenFunctionInJavaScriptResponse()
 
         // Then
         assertThat(alertsRaised, hasSize(1));
-        assertEquals("eval", alertsRaised.get(0).getEvidence());
+        assertEquals("eval(", alertsRaised.get(0).getEvidence());
```

Comment:
Maybe `startsWith` otherwise we are going to have to keep updating them each time.